modules/ROOT/pages/microservice-observability-metrics.adoc

```diff
@@ -223,10 +330,42 @@ For this example, the following response is generated from the REST endpoints:
 
 '''
 
+
 These types of metrics are available to add to your applications to make them observable.
 In production, operations teams can use these metrics to monitor the application, along with metrics that are automatically emitted from the JVM and the Open Liberty server runtime.
 If you're interested in learning more about using MicroProfile Metrics to build observability into your microservices, see the Open Liberty guide for link:/guides/microprofile-metrics.html[Providing metrics from a microservice].
 
+
+'''
+
+== Metric Tags
+
+Metrics instrumented with MicroProfile Metrics can be instrumented with key-value tags. The key is the tag name and the value can be referred to as the tag value. This allows for the instrumentation of multiple metrics, of the same type, using the same metric name as long as the set of tags have matching tag names, but with different values. If the metric types used or if the set of tag names do not match, then an `IllegalArgumentException` will be thrown. This is useful for logically grouping metrics that track similar statistics by using the same name and are then identified by their tags.
```

Comment:
this paragraph is really confusing, though i get the basic idea

Reply:
How about if it was trimmed down to:
```
Metrics instrumented with MicroProfile Metrics can be instrumented with tags. The tag consists of a name and value. This allows for the instrumentation of multiple metrics, of the same type, using the same metric name as long as the set of tags have matching names, but with different values. If the metric types used or if the set of tag names do not match, then an `IllegalArgumentException` will be thrown. This is useful for logically grouping metrics that track similar statistics by using the same name and are then identified by their tags.
```

Reply:
what if we lead with the benefit- something like:

> With MicroProfile metrics, you can logically group metrics that track similar statistics by instrumenting them with tags. Tags consist of a name and a value. You can instrument multiple metrics of the same type with tags of the same name but different values. However, in this configuration, if the metrics types are not the same or if the tag names do not match exactly, an `IllegalArgumentException` is thrown.